packages/web3-eth2-beaconchain/src/index.ts

```diff
@@ -36,7 +32,7 @@ export default class Web3Beacon {
      */
     async getGenesis(
         requestArguments?: Partial<Eth2RequestArguments>
-    ): Promise<RpcStringResult> {
+    ): Promise<{ [key: string]: any }> {
```

Comment:
@luu-alex `RpcStringResult` was used and is incorrect. I applied this bandaid solution, but these need to be replaced with properly typed responses for each method's return value